src/Symfony/Bundle/WebProfilerBundle/Csp/ContentSecurityPolicyHandler.php

```diff
@@ -146,6 +147,12 @@ private function updateCspHeaders(Response $response, array $nonces = array())
                 }
                 $headers[$header][$type][] = sprintf('\'nonce-%s\'', $nonces[$tokenName]);
             }
+
+            // unsafe-eval is only needed if the dumper was used
+            if (VarDumper::$dumped && !$this->authorizesEval($directives, 'script-src')) {
```

Comment:
would it make sense to always authorize unsafe-eval in the profiler?

Reply:
This is not about authorizing it in the profiler. It is about authorizing it in your project. Always authorizing `unsafe-eval` might hide places where your code violates your CSP rules, which would then be impossible to debug (as the dev environment would not have the same one).